### ISSUE TITLE
Clarify visibility and status settings in post-publish panel

### ISSUE THREAD

**Issue by sarahmonster** [WordPress/gutenberg#16127]
Scheduling posts is a source of confusion for users, often leading to accidentally breaking news embargoes (#13230) or other snafus (#12256). Since we're converting the pre-publish panel to use a modal (#15847), we have an opportunity to be more explicit here and use typographic hierarchy to improve understanding.

I'd like to suggest revealing more information in the pre-publish panel in order to clarify both the publish date and the visibility settings.

Publish date:
![Publish dates](https://user-images.githubusercontent.com/376315/58725788-f13c9f00-83d7-11e9-9062-135506c48c36.png)

Visibility:
![Publish visibilty](https://user-images.githubusercontent.com/376315/58726006-7e7ff380-83d8-11e9-80fa-e43d5e9736a8.png)

The wording on these will probably need some tweaking, as will the hierarchy, but providing additional checks and information here may help users make more informed decisions and avoid accidental publishing snafus.

I haven’t illustrated editing these controls yet (may hinge on results in #470), but we’ll want a way to edit the visibility and post date here as well.

I'd also recommend changing the wording on the publish button itself, depending on when the post is scheduled for:
- Future: Schedule post
- Past: Publish retroactively

Very related: #470

**Comment by lasmit** [comment]
I like the idea of fully writing out the description as you have done, as people rather read sentences than simplified words. Here's my addition to your idea (also taking [#470](https://github.com/WordPress/gutenberg/issues/470)) into account to make sure it's understandable with screen readers:

_Visibility_ 

This post will be visible to **anyone**

This post will be visible to **logged-in administrators only**

This post will be visible to **people with the password only**

In the shorthand after publishing this will then be more easy to understand. So instead of Visibility: **Public**, it will become Visible to **anyone**.

(My first github contribution, hope this helps)

**Comment by sarahmonster** [comment]
Thanks for the feedback, @lasmit! I like your suggestions here—they make the text a wee bit shorter, which is always good. I'm going to flag this for additional copy review as well, since I'm not sure if we should be opting for "your post" or "the post" here. (I don't have strong feelings either way!)

> In the shorthand after publishing this will then be more easy to understand. So instead of Visibility: Public, it will become Visible to anyone.

Can you clarify what you mean here? 🙂 I'd like to be sure this text is as clear as possible for users of all sorts of different  devices. 

We'll probably want to allow for editing the publish date and visibility settings directly from this dialog. In this case, I think it might make sense to have an edit control after the clarifying text, like so:

![Publish now (with edit)](https://user-images.githubusercontent.com/376315/60198701-24891700-983a-11e9-9022-e9a76f6fc24d.png)

That should help make the action super clear for screen readers, whilst still allowing us to use labels to better clarify the expected result.

**Comment by kristastevens** [comment]
Here are the copy changes for Publish Date I suggested in #16308, which make sense here: 

CASE: RETROACTIVE
Publish on
[retroactive date]
Your post will be published now and back-dated to **[date]**.

CASE: PRESENT
Publish on
[today's date]
Your post will be published **right now**.

CASE: FUTURE
Schedule for
[date]
Your post will be published **in X days**.

> flag this for additional copy review as well, since I'm not sure if we should be opting for "your post" or "the post" here.

I prefer "your" as it's a bit friendlier. 

As I noted in #16308, I'm partial to "Who can see my post" vs. Visibility as a header and I like @lasmit's suggestions about being super explicit about it, as noted: 

> This post will be visible to **anyone**
> 
> This post will be visible to **logged-in administrators only**
> 
> This post will be visible to **people with the password only**

**Comment by afercia** [comment]
Thanks for mentioning #470 (created on 20 Apr 2017...), as there are important accessibility considerations there strictly related to this issue.

I love to see typographic hierarchy used to improve this UI 💯 

> I've attempted a label change ("Publish" → "Schedule for"

Not sure I fully understand: do you mean the blue button? When you have a chance, could you please clarify or maybe point us to an example? Thanks!

**Comment by sarahmonster** [comment]
Thanks for the feedback here! Let's compare a few options:

A is the original options suggested:
![Publish date   visibility A](https://user-images.githubusercontent.com/376315/61046038-10195280-a3d4-11e9-9852-c29a506e5759.png)

B is the revised settings as per feedback:
![Publish date   visibility B](https://user-images.githubusercontent.com/376315/61046039-10195280-a3d4-11e9-9eae-afa6e588fbe5.png)

...and C is a suggested combination of the above, because who doesn't like more options 😉 :
![Publish date   visibility C](https://user-images.githubusercontent.com/376315/61046040-10195280-a3d4-11e9-9578-18a276bebf94.png)

Rationale for changes:
- "Who can see my post" feels a bit wordy. It seems it would be best here for us to match the labels used in #470, which will likely need to use "visibility" rather than "who can see my post", especially if translations are likely to make that string much longer.
- Using the current date in the "Present" case feels a bit heavy-handed, but it's probably helpful to have that in either the big display or the clarifying text below. Here, I'm opting for moving it to the clarifying text below because it seems like it would be easier to scan quickly & confirm that the settings are to your liking this way. (I don't always know what day & time it is, but I always know what "immediately" means.) I'm not opposed to switching these around though.
- In terms of "Publish" vs "Publish on", having balance between "Schedule for" and "Publish on" is nice, but the "on" seems like it can be dropped without any loss of meaning, whereas the same isn't true for "Schedule for". Optimising for shorter labels may be best here, especially if we want that text to read "Publish immediately" for the present (most common) case.
- I tried to simplify the visibility text clarifications a wee bit.

**Comment by karmatosed** [comment]
This has been discussed in the weekly design triage meetings and now is being closed to consider iterating later. The focus right now is on other areas which will likely see this area improved in maybe a different direction.
